frappe/desk/page/user_profile/user_profile.py

```diff
@@ -8,10 +8,10 @@ def get_energy_points_heatmap_data(user, date):
 		where
 			date(creation) > subdate('{date}', interval 1 year) and
 			date(creation) < subdate('{date}', interval -1 year) and
-			user = '{user}' and
+			user = %s and
 			type != 'Review'
 		group by date(creation)
-		order by creation asc""".format(user = user, date = date)))
+		order by creation asc""".format(date = date), user))
```

Comment:
Let's escape `date` too since it is a request parameter

Reply:
We can skip the date here... since its value is getting generated inside the method